indra/newview/llgltfmateriallist.cpp

```diff
@@ -288,6 +295,10 @@ class LLGLTFMaterialOverrideDispatchHandler : public LLDispatchHandler
                         else if (obj && obj->isAnySelected())
```

Comment:
Might be a good time to replace "obj->isAnySelected()" this with (getTE(results[i].mSide) && getTE(results[i].mSide)->isSelected()) to minimize update spam.